AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java

```diff
@@ -218,4 +228,74 @@ public FunctionItem createFromParcel(Parcel in) {
             void consume(AnkiActivity activity);
         }
     }
+
+    private static class ExceptionReportItem extends Item implements Parcelable{
```

Comment:
spacing, please consult our style guide https://github.com/ankidroid/Anki-Android/wiki/Code-style#white-spaces

```suggestion
    private static class ExceptionReportItem extends Item implements Parcelable {
```